VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistory.tsx

```diff
@@ -390,6 +391,39 @@ function PrescriptionHistory({ navigation, route }: PrescriptionHistoryProps) {
 
   const hasNoItems = filteredPrescriptions?.length === 0
 
+  const getNonVAMedsAlert = () => {
+    const linkProps: LinkProps = {
+      type: 'custom',
+      onPress: () => {
+        logAnalyticsEvent(Events.vama_webview(LINK_URL_MHV_VA_MEDICATIONS))
```

Comment:
Is it intentional to track this in addition to the events tracked by LinkWithAnalytics?